hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,4 @@
+RELEASE_TYPE: patch
+
+This patch add `LiveServerTestCase` and `StaticLiveServerTestCase` for django
+test.
```

Comment:
```suggestion
This release adds `LiveServerTestCase` and `StaticLiveServerTestCase` classes for django
tests.  Thanks to Ivan Tham for this feature!
```

And please also add yourself to the alphabetized list of contributors in `AUTHORS.rst` file!